server/etcdserver/server.go

```diff
@@ -2465,3 +2468,16 @@ func (s *EtcdServer) getTxPostLockInsideApplyHook() func() {
 func (s *EtcdServer) CorruptionChecker() CorruptionChecker {
 	return s.corruptionChecker
 }
+
+func (s *EtcdServer) buildLearnerMetric() {
```

Comment:
suggest to use `updateLearnerMetric` here

Reply:
updated